### PR TITLE
Add release notes for ML.NET 0.7

### PR DESCRIPTION
**Pull request by GalOshri**
This adds release notes for ML.NET 0.7

### CODE REVIEW

File: docs/release-notes/0.7/release-0.7.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# ML.NET 0.7 Release Notes
+
+Today we are excited to release ML.NET 0.7, which our algorithms strongly
+recommend you to try out! This release enables making recommendations with
```
Comment (justinormont) [inline docs/release-notes/0.7/release-0.7.md]: Was there a probability associated with this recommendation/prediction? We should create docs showing how to train this model using the new `Matrix Factorization` to create a recommendation model.

File: docs/release-notes/0.7/release-0.7.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      [#1098](https://github.com/dotnet/machinelearning/issues/1098).
+    * Example usage can be found
+      [here](https://github.com/dotnet/machinelearning/blob/d68388a1c9994a5b429b194b64b2b0782834cb78/docs/code/MlNetCookBook.md)
+
+* Open sourced [NimbusML](https://github.com/microsoft/nimbusml): experimental
+  Python bindings for ML.NET. 
+
+    * Some teams at Microsoft found it useful to use ML.NET capabilities in
+      Python environments. NimbusML provides Python APIs to ML.NET and easily
+      integrates into [Scikit-Learn](http://scikit-learn.org/stable/)
```
Comment (justinormont) [inline docs/release-notes/0.7/release-0.7.md]: _[not a bug]_
Seems scikit-learn does not have a http_s_ website. 

@amueller, as always I prefer using https links; any possibility of getting scikit-learn's website on TLS?

Comment (justinormont) [inline docs/release-notes/0.7/release-0.7.md]: Scikit-learn is generally lowercase, except when at the start of a sentence. 

```suggestion
      integrates into [scikit-Learn](http://scikit-learn.org/stable/)
```

Comment (justinormont) [reply docs/release-notes/0.7/release-0.7.md]: I see the TLS cert work is in progress -- https://github.com/scikit-learn/scikit-learn/issues/12278

Comment (amueller) [reply docs/release-notes/0.7/release-0.7.md]: yeah @ogrisel needs to have a look.

File: docs/release-notes/0.7/release-0.7.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      `BinaryClassification`/`Regression`, various transforms/trainers, and
+      evaluation. More information can be found in
+      [#1098](https://github.com/dotnet/machinelearning/issues/1098).
+    * Example usage can be found
+      [here](https://github.com/dotnet/machinelearning/blob/d68388a1c9994a5b429b194b64b2b0782834cb78/docs/code/MlNetCookBook.md)
+
+* Open sourced [NimbusML](https://github.com/microsoft/nimbusml): experimental
+  Python bindings for ML.NET. 
+
+    * Some teams at Microsoft found it useful to use ML.NET capabilities in
```
Comment (chris-lauren) [inline docs/release-notes/0.7/release-0.7.md]: Would re-word to indicate that NimbusML makes it easy for data scientists to train models in Python and hand them off to .NET developers to include in their apps and services using model scoring in ML.NET

Comment (GalOshri) [reply docs/release-notes/0.7/release-0.7.md]: Thanks! Addressed.

File: docs/release-notes/0.7/release-0.7.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+* Open sourced [NimbusML](https://github.com/microsoft/nimbusml): experimental
+  Python bindings for ML.NET. 
+
+    * NimbusML makes it easy for data scientists to train models in Python and
+      hand them off to .NET developers to include in their apps and services
+      using ML.NET. 
+    * NimbusML components easily integrate into
+      [scikit-learn](http://scikit-learn.org/stable/) pipelines. 
+    * Note that NimbusML is an experimental project without the same support
+      level as ML.NET.
```
Comment (justinormont) [inline docs/release-notes/0.7/release-0.7.md]: Did we want a "Changes" section? Perhaps listing the renamed/re-name-spaced components. If we're lucky, we'll help a single dev, lucky enough to read the note, avoid a few minutes of confusion.

File: docs/release-notes/0.7/release-0.7.md (hunk 2 above)
Comment (ogrisel) [reply docs/release-notes/0.7/release-0.7.md]: Fixed!

**Pull request merged**